src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -237,6 +237,9 @@ export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElemen
   ]);
 };
 
-export const displayParticipantCount = (count: number): string => {
+export const displayParticipantCount = (count: number | undefined): string => {
+  if (count === undefined) {
+    return '';
+  }
   return count <= 19 && count > 0 ? 'Less than 20' : count.toString();
```

Comment:
This should be `=== 19` to make it clear that the count will either be `19`, `0`, or some value >= `20`. We won't get any value back between `0` and `19`.
```suggestion
  return count === 19 && count > 0 ? 'Less than 20' : count.toString();
```